flattentool/cli.py

```diff
@@ -180,6 +188,10 @@ def create_parser():
         "--convert-titles",
         action='store_true',
         help="Convert titles. Requires a schema to be specified.")
+    parser_unflatten.add_argument(
```

Comment:
Should not be relevant to unflatten as it uses sub sheet names from the actual spreadheet.